router/batchrouter/asyncdestinationmanager/bing-ads/bingads_test.go

```diff
@@ -417,96 +466,61 @@ var _ = Describe("Bing ads", func() {
 			initBingads()
 			ctrl := gomock.NewController(GinkgoT())
 			bingAdsService := mock_bulkservice.NewMockBulkServiceI(ctrl)
-			templateFilePath := filepath.Join(currentDir, "test-files/uploadstatus.csv") // Path of the source file
+			templateFilePath := filepath.Join(currentDir, "test-files/status-check.zip") // Path of the source file
 			testFilePath := filepath.Join(currentDir, "test-files/test_copy.csv")        // Path of the destination folder
-
+			defer os.Remove("test-files/test_copy.csv")
 			err := DuplicateFile(templateFilePath, testFilePath)
```

Comment:
do we really need this, we are sending file from mock server, why will this file be deleted?

Reply:
After completing the test it is not getting removed. So I removed it manually.